examples/numbersapi.py

```diff
@@ -0,0 +1,49 @@
+__requires__ = "httpx"
+
+import asyncio
+from pprint import pp
+from random import choices, randint
+
+import httpx
+from aiotinydb import AIOTinyDB
+from tinydb import where
+
+
+async def main():
+    numbers = {
+        f"{randint(1,12)}/{randint(1,31)}"
+        if tp == "date"
+        else randint(1, 2000): tp
+        for tp in choices(["trivia", "math", "date", "year"], k=10)
+    }
+
+    async with (
+        httpx.AsyncClient() as http_client,
+        AIOTinyDB("numbers.json") as db,
+    ):
+
+        async def fetch_number(number, info_type):
+            r = await http_client.get(
+                f"http://numbersapi.com/{number}/{info_type}"
```

Comment:
Can we somehow pull this source locally?

Reply:
I wanted to add an example that shows a "real world scenario" where using non-blocking IO is beneficial. Using external HTTP requests seems to be one of the most common use cases. We could also replace it with some fake coroutine that awaits `asyncio.sleep()`

However, I see this example totally misses that point because the database is only written to disk once.